animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -141,13 +154,18 @@ cdef class Mesh:
                 Vertices: {}
                 Edges: {}
```

Comment:
Try replacing this with an f-string.

Reply:
Omar, I have checked with the fresh AN compile build, there is no `runtime` warning. It is working fine.

Reply:
Good. To clarify, I wanted you to using python's f-strings here. See https://www.python.org/dev/peps/pep-0498/